index.html

```diff
@@ -0,0 +1,137 @@
+        </div>
+        <div class="aboutMeBoxes">
+            <button class="interests" type="checkbox">INTERESTS</button>
+            <button class="qualifications">QUALIFICATIONS & ACCOLADES</button>
+            <button class="something">SOMETHING ELSE ABOUT ME</button>
+            <a class="anchor" id="portfolio"></a>
+        </div>
+    </div>
+    <div class="portfolioText">
+        <h1>PORTFOLIO</h1>
+        <sub>A Look At My Projects</sub>
+        <div class="spacer"></div>
+    </div>
+    <div class="portfolio">
+        <div class="projectBlocks">
+            <!--All commented links will be images rather than text (lines 67 through 107 where relevant-->
+            <div class="projectOne">
+                <div class="project projectBox">
+                    <a href="file:///Users/academy/Documents/Projects/PilotShop/pilotshop.html">
+                        <img class="projectPics" src="pilotshop.png">
+                    </a>
+                </div>
+                <div class="projectSummary projectBox">Summary One</div>
+            </div>
+            <div class="projectTwo">
+                <div class="project projectBox">
+                    <!--<a href="url to project 2">-->
+                       Project Two
+                    <!--</a>-->
+                </div>
+                <div class="projectSummary projectBox">Summary Two</div>
+            </div>
+            <div class="projectThree">
+                <div class="project projectBox">
+                    <!--<a href="url to project 3">-->
+                        Project Three
+                    <!--</a>-->
+                </div>
+                <div class="projectSummary projectBox">Summary Three</div>
+            </div>
+            <div class="projectFour">
+                <div class="project projectBox">
+                    <!--<a href="url to project 4">-->
+                        Project Four
+                    <!--</a>-->
+                </div>
+                <div class="projectSummary projectBox">Summary Four</div>
+            </div>
+            <div class="projectFive">
+                <div class="project projectBox">
+                    <!--<a href="url to project 5">-->
+                        Project Five
+                    <!--</a>-->
+                </div>
+                <div class="projectSummary projectBox">Summary Five</div>
+            </div>
+            <div class="projectSix">
+                <div class="project projectBox">
+                    <!--<a href="url to project 6">-->
+                        Project Six
+                    <!--</a>-->
+                </div>
+                <div class="projectSummary projectBox">Summary Six</div>
+            </div>
+        </div>
+    </div>
+<!--<footer>-->
```

Comment:
will you put in a footer? if not why is it here? if you're can you move forward without footer?

not good to have too much code commented out.

Reply:
all commented out as it is design focused and doesn't concern page layout. have written comments above those parts of code explaining why they are currently commented out and stated this also in my comments at the top of the pull request......